modules/userId/index.js

```diff
@@ -160,6 +167,24 @@ function getStoredValue(storage) {
   return storedValue;
 }
 
+/**
+ * @param {SubmoduleStorage} storage
+ * @returns {date} browser date and time of the last storage
+ */
+function getStoredDate(storage) {
```

Comment:
Can you use `getStoredValue` to avoid adding another function `getStoredDate`?

Reply:
Sure, here is a proposal